scope3ai/tracers/openai/speech_to_text.py

```diff
@@ -23,11 +24,11 @@ class AnnotatedStr(str):
     scope3ai: Optional[Scope3AIContext] = None
 
 
-class Transcription(_Transcription):
+class Transcription(BaseModelResponse, _Transcription):
```

Comment:
To me overloading the model_dump to remove scope3ai because somehow it's not compatible feels hacky. Because there won't be a possible way to dump the whole response then ?

Ok for now, but let's try to revisit later this part.